README.md

```diff
@@ -154,6 +156,15 @@ sudo cp target/release/alacritty /usr/local/bin # or anywhere else in $PATH
 cp Alacritty.desktop ~/.local/share/applications
 ```
 
+#### OS X
+
+To install the desktop entry for Alacritty, run
+
+```sh
+make app
```

Comment:
@brycefisher I'm leaning towards adding it behind a feature gate. Folks who don't want it can have it 100% compiled out and not affect perf, and those that are interested can opt-in. Rust's feature system would make it pretty easy.